controllers/bookings.js

```diff
@@ -0,0 +1,18 @@
+const db = require('../models');
+
+module.exports = {
+  get: (req, res, next) => {
+    db.Booking.find({})
+      .exec()
+      .then(data => {
+        if (!data) {
```

Comment:
Booking.find passes a `err` and `array` into our call back function. `if(!data)` if data is a array might now work